modules/migration-installing-mtc-on-ocp-4.adoc

```diff
@@ -27,7 +27,5 @@ endif::[]
 +
 On the *Installed Operators* page, the *{mtc-full} Operator* appears in the *openshift-migration* project with the status *Succeeded*.
 
-. Click *{mtc-full} Operator*.
```

Comment:
These steps should not be removed from installation document. Should only be removed from upgrade documents.